attack-theme/static/style/_colors.scss

```diff
@@ -15,7 +22,7 @@ $attack-colors: (
  * cannot have inner text.
  */
 $colors: (
-    primary: (color: map-get($attack-colors, attack-orange), on-color: white),  //used for header and some nav elements
+    primary: (color: if(variable-exists($name: primary_override), map-get($attack-colors, attack-orange), map-get($user-colors, user-gray)),  on-color: white),  //used for header and some nav elements
     secondary: (color: map-get($attack-colors, attack-blue), on-color: white),  //used for footer and some buttons
```

Comment:
We should consider making the secondary color also the dark gray color when not using the ATT&CK brand. MITRE Blue is too close to the gray and therefore it looks weird when shown together, plus we should probably remove it anyway since it's also a brand color.

AFAIK this will affect only the footer and navigator layer links on group/software pages, both of which are fine to change to the dark gray.